go/vt/wrangler/schema.go

```diff
@@ -20,8 +20,8 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"html/template"
 	"sync"
+	"text/template"
```

Comment:
This is used for SQL schemas, so we really should be using `text/template` here as it's not HTML.